tests/txt.test.js

```diff
@@ -393,3 +394,35 @@ describe('validateCodeBlockLicenses', () => {
     ])
   })
 })
+
+describe('The document has more than 15 pages and not Table of Contents.', () => {
+  test('Table of Contents exists and pages less 15', async () => {
+    const doc = cloneDeep(baseTXTDoc)
+
+    doc.data.possibleIssues.isTableOfContentsExists = true
+    doc.data.pageCount = 14
+
+    await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.NORMAL })).resolves.toHaveLength(0)
+    await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.FORGIVE_CHECKLIST })).resolves.toHaveLength(0)
+    await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.SUBMISSION })).resolves.toHaveLength(0)
+  })
+  test('Table of Contents missing and pages less 15', async () => {
+    const doc = cloneDeep(baseTXTDoc)
+    doc.data.possibleIssues.isTableOfContentsExists = false
+    doc.data.pageCount = 14
+
+    await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.NORMAL })).resolves.toContainError('DOCUMENT_HAVE_MORE_15_PAGES_OR_MISS_TABLE_OF_CONTENTS', ValidationError)
+    await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.FORGIVE_CHECKLIST })).resolves.toContainError('DOCUMENT_HAVE_MORE_15_PAGES_OR_MISS_TABLE_OF_CONTENTS', ValidationError)
+    await expect(validateTableOfContentsAndDocumentPages(doc, { mode: MODES.SUBMISSION })).resolves.toContainError('DOCUMENT_HAVE_MORE_15_PAGES_OR_MISS_TABLE_OF_CONTENTS', ValidationWarning)
+  })
```

Comment:
This is incorrect. As long as we're keeping the page theshold concept, there should be no error if there is no table of contents for documents with fewer than the threshold number of pages.